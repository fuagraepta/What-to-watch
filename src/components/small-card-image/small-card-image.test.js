import React from 'react';
import renderer from 'react-test-renderer';

import SmallCardImage from './small-card-image.jsx';

describe(`The component is rendered correctly`, () => {
  it(`Video player correctly renders with transferred mock data`, () => {
    const videoPlayer = renderer
    .create(
        <SmallCardImage
          renderVideo = {jest.fn()}
        />
    )
    .toJSON();

    expect(videoPlayer).toMatchSnapshot();
  });
});
