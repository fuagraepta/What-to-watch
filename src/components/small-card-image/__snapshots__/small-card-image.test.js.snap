// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly Video player correctly renders with transferred mock data 1`] = `
<div
  className="small-movie-card__image"
/>
`;
